AcademyApp/ViewControllers/LecturesViewController.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  LecturesViewController.swift
+//  AcademyApp
+//
+//  Created by Peter Sevcik on 19/06/2019.
+//  Copyright © 2019 Peter Sevcik. All rights reserved.
+//
+
+import UIKit
+
+class LecturesViewController: UIViewController {
+
+    var lectures = ["a", "b", "c", "d", "e", "f"]
+
+    @IBOutlet private weak var lecturesLabel: UILabel!
+    @IBOutlet private weak var collectionView: UICollectionView!
+    @IBOutlet private weak var collectionViewLeadingConstraint: NSLayoutConstraint!
+    @IBOutlet private weak var collectionViewTrailingConstraint: NSLayoutConstraint!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setup()
+    }
+}
+
+extension LecturesViewController: UICollectionViewDataSource {
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return lectures.count
+    }
+
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+
+        let cell: LecturesCollectionViewCell = collectionView.dequeReusableCell(for: indexPath)
+        cell.configure()
+        return cell
+    }
+}
+
+extension LecturesViewController: UICollectionViewDelegate {
+
+}
+
```

Comment:
as discussed remake pls to table view

Reply:
Remaked collectionView to be a tableView.